components/riemann/scripts/creation_validation.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
The name `creation_validation` is misleading.. as you're not actually validating the creation of anything. I'd call it upgrade-validation (note the dash instead of the underscore as it's less acceptable in modules.)

WDYT?